src/manifold/include/manifold.h

```diff
@@ -234,6 +234,7 @@ class Manifold {
   std::pair<Manifold, Manifold> SplitByPlane(glm::vec3 normal,
                                              float originOffset) const;
   Manifold TrimByPlane(glm::vec3 normal, float originOffset) const;
+  Manifold Minkowski(const Manifold&, bool inset = false);
```

Comment:
Thanks, and a further refinement - I try to avoid boolean arguments in our public API. Let's make two functions instead: `MinkowskiSum` and `MinkowskiDifference`.

Reply:
Oh, and this should be a `const` method like the others.

Reply:
Sure; out of curiosity, what's wrong with them?

Reply:
It's a good piece of API readability advice I heard some years back. It's a lot more clear to come across `cube.MinkowskiDifference(sphere)` than `cube.Minkowski(sphere, true)`.

Reply:
Also, worth noting, `Minkowski Difference` is a well-defined term meaning to do a normal Minkowski Sum, but with a mesh that has been scaled by -1.

This is frequently used in collision detection and its the core of the GJK algorithm; if the Minkowski Difference of two meshes encompass the origin, then those two meshes are colliding; there's a great intro by Casey Muratori here: https://youtu.be/SDS5gLSiLg0?t=2685

Maybe another name makes sense... `MinkowskiAddition`/`MinkowskiSubtraction` or `MinkowskiAdd`/`MinkowskiSubtract`, maybe?

Reply:
Also, a cool gif I made to illustrate this a while ago:
![Minkowski Difference](https://camo.githubusercontent.com/b6cf57f94766243c162d3932a2e10e38a329feb00792c5f7fc2639c561f479e6/68747470733a2f2f692e696d6775722e636f6d2f5a6d415433536d2e676966)

The origin is that white pixel on the bottom right; sorry it's kind of hard to see through the grainy gif encoding 😅

Reply:
Trying it out with `MinkowskiAdd`/`MinkowskiSubtract`, to make the distinction from the GJK operation clearer.

Reply:
Thanks for the talk link, that was cool. However, I think the operation we're talking about really is the Minkowski Difference: https://en.wikipedia.org/wiki/Minkowski_addition; the collision detection one is a little different.

Reply:
Hrmm, the papers for GJK do mention that the term is overloaded; this paper addresses it briefly in Section 3.4:
https://www.medien.ifi.lmu.de/lehre/ss10/ps/Ausarbeitung_Beispiel.pdf

Given that `MinkowskiDifference` is overloaded, would you rather have `MinkowskiSum`/`MinkowskiDifference` or `MinkowskiAdd`/`MinkowskiSubtract`?

Reply:
Let's go with Sum and Difference - I think those are clearer for this context. We can mention in the docs that they correspond to erode and dilate, which I think will remove any confusion.

Reply:
Alrighty; done 👍